### DISCUSSION TITLE
Using component as displayValue on ComboBoxInput

### DISCUSSION THREAD

**Discussion by d8vjork** [tailwindlabs/headlessui#2008] [Ideas]
So I've a project where I decorated the options list of my ComboBox with icons, problem thought is that only works with strings

Will be awesome if we can return a component instead and then headlessui at its end uses `<component :is="displayUsing(val)">` whenever `displayUsing` returns something else than an string

<img width="439" alt="image" src="https://user-images.githubusercontent.com/2331052/201160066-98eed7cc-6e98-47b9-8f78-d442aebe8623.png">

**Comment by HugoCamposLopez** [comment]
If you find something. let me know.
